### PR TITLE
[CX-938] replacing admonitions with raw HTML

### PR DESCRIPTION
**Pull request by MostFrumiousBandersnatch**
image

### CODE REVIEW

File: docs/integrations/orchestration/dbt_cloud/prerequisites.md [hunk 1]
```diff
@@ -34,10 +34,11 @@ dbt Cloud CI requires you to create at least two dbt Cloud jobs: a Production jo
 
 We recommend creating a job in dbt Cloud that sends production artifacts (the `manifest.json` file) to Datafold on a regular basis. This is the easiest way to get started with Datafold.
 
-:::info Continuous Deployment
-Alternatively, you can set up continuous deployment by
-[creating a Merge Trigger Production Job](../../../guides/ci_guides/dbt_cloud.md#merge-trigger-production-job) using a tool like GitHub Actions.
-:::
+<div style={{backgroundColor: '#e6f4ff', border: '1px solid #91caff', borderRadius: '8px', padding: '8px 12px'}}>
```
Comment (dlawin) [inline docs/integrations/orchestration/dbt_cloud/prerequisites.md:37]: Screenshot 2023-03-09 at 11 17 49 AM

This one looks a little odd.

We should add `Alternatively, you can set up continuous deployment by` back, I think we can drop the `info Continuous Deployment`

Comment (leoebfolsom) [reply docs/integrations/orchestration/dbt_cloud/prerequisites.md:37]: FYI I concurrently complained about this same thing in Linear (I was asked by Anastasia to review the experience on testing.datafoldio ) https://linear.app/datafold/issue/CX-161#comment-4e20886c

**Comment by MostFrumiousBandersnatch** [comment]
@leoebfolsom @dlawin Sorry for late reply.
 I suppose that this task does no longer make sense, at least that kind of sense it was initially about. 
It's appeared very troublesome to maintain the markdown both rich enough to deserve to be present on our doc site and simple enough to be digested correctly by our embedded doc viewer.
Thus, instead of merging this PR, i suggest to maintain two diverged versions, and sync them from time to time.

File: docs/integrations/orchestration/dbt_cloud/prerequisites.md (hunk 1 above)
Comment (MostFrumiousBandersnatch) [reply docs/integrations/orchestration/dbt_cloud/prerequisites.md:37]: Done.
image

**Comment by leoebfolsom** [comment]
@MostFrumiousBandersnatch I agree with your comments from May 5 -- what are your current thoughts on this PR? Should it be closed?

**Pull request closed without merging**